### DISCUSSION TITLE
Is it planned to improve the plugin for migrating data about the virtual machine, such as its ip address?

### DISCUSSION THREAD

**Discussion by mrkobz** [emersonfelipesp/netbox-proxbox#70] [General]
Hello. Is it planned to improve the plugin for migrating data about the virtual machine, such as its ip address?

**Comment by emersonfelipesp** [comment]
> Hello. Is it planned to improve the plugin for migrating data about the virtual machine, such as its ip address?

Hello @mrkobz! I plan to import everything in a way that only Proxmox backend becomes necessary, and all CRUD actions could be done in NetBox instead of Proxmox GUI.

Also, it was already deployed network/IP address integration and Proxbox is gathering info from Proxmox. Could you please test it?
I'm closing this discussion, since it is already solved!
